gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/ClassUniquenessAnalyzer.java

```diff
@@ -60,43 +56,26 @@ public void analyzeConfiguration(Configuration configuration) {
         Map<String, Set<ModuleVersionIdentifier>> classToJars = new HashMap<>();
         Map<String, Set<HashCode>> tempClassToHashCodes = new HashMap<>();
```

Comment:
Any reason why we don't use a `SetMultimap` here as well? That way we also could get rid of the `multiMapPut` method.

Reply:
I added this refactor to the PR. The section that looks for singleton-set values requires an `asMap()`, but that's the only wrinkle.